packages/tinacms/src/components/TinaProvider.tsx

```diff
@@ -40,6 +40,7 @@ export const TinaProvider: React.FC<TinaProviderProps> = ({
   return (
     <CMSContext.Provider value={cms}>
       <ModalProvider>
+        <FontLoader active={true} />
```

Comment:
Would it be completely against the intended use of the plugin system if you used it to configure or optionally replace the `FontLoader`?